### ISSUE TITLE
finalise iso_week implementation for 1.1.0 release

### ISSUE THREAD

**Issue by thibautjombart** [reconhub/incidence#17]
@caijun check that all your commits are indeed there - they should be

I have added a few tests for `iso_weeks` but bits and pieces are still missing. 
I have:
```
> covr::package_coverage()
incidence Coverage: 99.24%
R/conversion.R: 95.00%
R/incidence.R: 98.40%
R/fit.R: 100.00%
R/palettes.R: 100.00%
R/plot.R: 100.00%
R/pool.R: 100.00%
R/subset.R: 100.00%
```

For details, check:
```
covr::shine(covr::package_coverage())
```

**Comment by thibautjombart** [comment]
@caijun  I have just added you as contributor to the github project

**Comment by caijun** [comment]
@thibautjombart I just checked the commits histories of my incidence fork and your incidence master branch.

![image](https://cloud.githubusercontent.com/assets/2522593/21043317/41dba196-be31-11e6-998d-f4d5fc1b5628.png)

![image](https://cloud.githubusercontent.com/assets/2522593/21043322/46f48972-be31-11e6-87ac-60ea73684a1e.png)

You see, my commits after Nov 28 haven't been merged into the master branch since you updated the master branch on Dec 3 and our commits have conflicts with your commits in the master branch. You can see those conflicts in the newest PR https://github.com/reconhub/incidence/pull/18 .

![image](https://cloud.githubusercontent.com/assets/2522593/21043411/bfab7a88-be31-11e6-9080-2b36c6aa41f7.png)

**Comment by thibautjombart** [comment]
Right, I'll have a look.

**Comment by thibautjombart** [comment]
OK, so as of de82278 everything should be put together. I haven't seen conflicts when doing the merge, oddly. Let me know if it looks okay.

**Comment by thibautjombart** [comment]
If it's all fine, all we need is get back to full coverage. There's just a couple of cases not tested, would be great if you could have a look?

```
 > covr::package_coverage()
incidence Coverage: 99.24%
R/conversion.R: 95.00%
R/incidence.R: 98.40%
R/fit.R: 100.00%
R/palettes.R: 100.00%
R/plot.R: 100.00%
R/pool.R: 100.00%
R/subset.R: 100.00%
```

**Comment by caijun** [comment]
But in the commits of master branch, I still can't my commits after Nov 28. In my commits, I have updated the x axis tick labels from dates to isoweeks for some figures in the README.md. See the README.md in my fork.

https://github.com/caijun/incidence

**Comment by caijun** [comment]
Also you can see the implementation of plot.incidence() in the master branch, which is without argument `labels_iso_week`.

**Comment by thibautjombart** [comment]
I see. Odd. These commits are not on the origin/isoweeks so they never made it out of your fork. Something went wrong with the PR or its merging, which is odd as when merging I picked your versions over the others. 

Now that you are on the project, can you try pulling the current isoweek, and merging / pushing your version directly (without PR)?

**Comment by caijun** [comment]
@thibautjombart I have fixed the conflicts. Now the isoweek branch has included all my commits. You can merge it into master after you update the websites.

**Comment by thibautjombart** [comment]
Should be all good now as of 8c27c4fad16f04b7e27e06ac1b198469ca403824
In terms of coverage I get:
```
> covr::package_coverage()
incidence Coverage: 97.28%
R/plot.R: 88.41%
R/conversion.R: 94.74%
R/incidence.R: 98.40%
R/fit.R: 100.00%
R/palettes.R: 100.00%
R/pool.R: 100.00%
R/subset.R: 100.00%
```

**Comment by caijun** [comment]
Great. I think we need more tests to increase the test coverage to 100%.

**Comment by thibautjombart** [comment]
Yep. You happy add them? Once this is done I think we're good to release on CRAN.

**Comment by caijun** [comment]
OK. I will try.

**Comment by thibautjombart** [comment]
Great, thanks!
